schemas/appeal-s78.schema.json

```diff
@@ -0,0 +1,1078 @@
+      "$comment": "Only available in S78 cases"
+    },
+    "eiaColumnTwoThreshold": {
+      "type": ["boolean", "null"],
+      "description": "EIA specific threshold",
+      "$comment": "Only available in S78 cases"
+    },
+    "eiaScreeningOpinion": {
+      "type": ["boolean", "null"],
+      "description": "EIA screening opinion",
+      "$comment": "Only available in S78 cases"
+    },
+    "eiaRequiresEnvironmentalStatement": {
+      "type": ["boolean", "null"],
+      "description": "EIA environmental statement required",
+      "$comment": "Only available in S78 cases"
+    },
+    "eiaCompletedEnvironmentalStatement": {
+      "type": ["boolean", "null"],
+      "description": "EIA completed environmental statement required",
+      "$comment": "Only available in S78 cases"
+    },
+    "eiaHasStatutoryConsultees": {
+      "type": ["boolean", "null"],
+      "description": "EIA statutory consultees",
+      "$comment": "Only available in S78 cases"
+    },
+    "eiaConsultedBodiesDetails": {
+      "type": ["string", "null"],
+      "description": "The details of the bodies consulted by EIA"
+    },
+    "eiaHasConsultationResponses": {
+      "type": ["boolean", "null"],
+      "description": "EIA consultation responses",
+      "$comment": "Only available in S78 cases"
+    },
+    "hasEmergingPlan": {
+      "type": ["boolean", "null"],
+      "description": "Indicates emerging plans",
+      "$comment": "Only available in S78 cases"
+    },
+    "hasSupplementaryPlanningDocs": {
+      "type": ["boolean", "null"],
+      "description": "Indicates supplementary planning documents",
+      "$comment": "Only available in S78 cases"
+    },
+    "hasInfrastructureLevy": {
+      "type": ["boolean", "null"],
+      "description": "Indicates the existence of an infrastructure levy",
+      "$comment": "Only available in S78 cases"
+    },
+    "isInfrastructureLevyFormallyAdopted": {
+      "type": ["boolean", "null"],
+      "description": "Indicates if the infrastructure levy is formally adopted",
+      "$comment": "Only available in S78 cases"
+    },
+    "infrastructureLevyAdoptedDate": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "description": "The date of the infrastructure levy adoption",
+      "$comment": "Only available in S78 cases"
+    },
+    "infrastructureLevyExpectedDate": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "description": "The expected date of the infrastructure levy",
+      "$comment": "Only available in S78 cases"
+    },
+    "lpaProcedurePreference": {
+      "type": ["string", "null"],
+      "enum": ["written", "hearing", "inquiry", null],
+      "description": "The procedure preference indicated by the LPA",
+      "$comment": "Only available in S78 cases"
+    },
+    "lpaPreferHearingDetails": {
```

Comment:
same opinion here could this be:
lpaProcedurePreference
lpaProcedurePreferenceDetails
lpaPreferenceDuration